lib/diagnostic.rb

```diff
@@ -31,31 +31,31 @@
 
 character = 'Jar Jar Binks'
 
-Diagnostic.interpolation = 'ANSWER is really a Sith Lord.'
+Diagnostic.interpolation = character + 'is really a Sith Lord.'
 
```

Comment:
Nice! This is definitely a valid way of interpolating strings in Ruby!
A more common way would be `"#{character} is really a Sith Lord."`